WultraMobileTokenSDK/Operations/Model/UserOperation/WMTResultTexts.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2024 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Texts for the result of the operation.
+///
+/// This includes messages for different outcomes of the operation such as success, rejection, and failure.
+public class WMTResultTexts: Codable {
```

Comment:
Missing `init`. Without the init, the `WMTOperationFormData.init` method does not make sense since it's now possible to pass an object that can't be created...

Reply:
🤦‍♂️ tru dat